File: client/src/models/Boss.js
export class Boss {
  constructor(data) {
    this.id = data.id;
    this.name = data.name;
    this.img = data.img;
    this.damage = data.damage;
    this.health = data.health;
    this.creatorId = data.creatorId;
  }
  get backgroundStyle() {
    return {
      backgroundImage: `url(${this.img})`
    }
  }
}

File: server/Models/Boss.cs
namespace bossmonster.Models;

public class Boss : RepoItem<int>
{
    public string Name { get; set; }
    public string Img { get; set; }
    public int Damage { get; set; }
    public int Health { get; set; }
    public string CreatorId { get; set; }
    public Profile Creator { get; set; }

}
